common/views/components/Modal/Modal.js

```diff
@@ -134,8 +193,8 @@ const Modal = ({ children, isActive, setIsActive, width = null }: Props) => {
   return (
     <>
       {isActive && <Overlay onClick={() => setIsActive(false)} />}
-      {isActive && (
-        <ModalWindow ref={modalRef} width={width}>
+      <CSSTransition in={isActive} classNames="fade" timeout={350}>
```

Comment:
What are we doing about [reduce motion](https://a11y-101.com/development/reduced-motion)? There is so much good work on the accessibility here, it'd be good to pay attention to that.

Reply:
Good shout, I've added this